iceoryx_posh/test/moduletests/test_roudi_process_introspection.cpp

```diff
@@ -204,7 +204,7 @@ TEST_F(ProcessIntrospection_test, addRemoveNode)
         const char NODE_2[] = "the_octagon";
```

Comment:
Please adjust year in the header.

Reply:
I think this is not necessary since I just replaced the word ``runnable`` by ``node`` in some comments.